app/models/solidus_avatax_certified/line.rb

```diff
@@ -28,7 +28,7 @@ def item_line(line_item)
         itemCode: line_item.variant.sku,
         quantity: line_item.quantity,
         amount: line_item.amount.to_f,
-        discounted: discounted?(line_item),
+        discounted: true,
```

Comment:
@ashishtajane Do we need to check if order has discount for this?
```suggestion
        discounted: discounted?(order),
```